### PR TITLE
blockdev_commit_hotunplug:update expected job status

### PR DESCRIPTION
**Pull request by aliang123**
Update expected job status from running to running or ready

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2127794

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_hotunplug.q35: PASS (100.94 s)

**Comment by aliang123** [comment]
@zhencliu @MiriamDeng or @maxujun, please help to review, thanks.

File: qemu/tests/blockdev_commit_hotunplug.py [hunk 1]
```diff
@@ -28,11 +28,17 @@ def commit_snapshots(self):
                                        timeout=60, step=1.0)
         if not unplug_s:
             self.test.fail("Hotunplug device failed")
-        job_status = str(job_utils.query_block_jobs(self.main_vm))
-        if self.params["expect_status"] not in job_status:
-            self.test.fail("Job status not correct,job status is %s"
-                           % job_status)
         job_id = args.get("job-id", device)
+        block_jobs = job_utils.query_block_jobs(self.main_vm)
```
[the remark this answers is not in the document]
Comment (aliang123) [reply qemu/tests/blockdev_commit_hotunplug.py]: Have updated it to jobs = job_utils.get_block_job_by_id(self.main_vm, job_id), @zhencliu

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_hotunplug.q35: PASS (48.05 s)

File: qemu/tests/blockdev_commit_hotunplug.py [hunk 2]
```diff
@@ -28,11 +28,17 @@ def commit_snapshots(self):
...
-                           % job_status)
         job_id = args.get("job-id", device)
+        jobs = job_utils.query_block_jobs(self.main_vm)
+        for job in jobs:
+            if job["device"] == job_id:
+                if job["status"] in self.params["expect_status"]:
+                    break
+                else:
+                    self.test.fail("Job status not correct,job status is %s"
+                                   % job)
```
Comment (maxujun) [inline qemu/tests/blockdev_commit_hotunplug.py:39]: Here should be job["status"],right?

Comment (aliang123) [reply qemu/tests/blockdev_commit_hotunplug.py]: have updated, thanks @maxujun.

File: qemu/tests/blockdev_commit_hotunplug.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhencliu) [reply qemu/tests/blockdev_commit_hotunplug.py]: hi @aliang123 , it looks that this part of code is not changed yet, would you please make a double check? Thanks.

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_hotunplug.q35: PASS (44.41 s)

**Comment by zhencliu** [comment]
LGTM

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_hotunplug.q35: PASS (42.21 s)

**Comment by maxujun** [comment]
LGTM

(1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_commit_hotunplug: PASS (89.35 s)

**Pull request merged**